specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/virtualNetwork.json

```diff
@@ -1702,6 +1702,10 @@
           "$ref": "#/definitions/DhcpOptions",
           "description": "The dhcpOptions that contains an array of DNS servers available to VMs deployed in the virtual network."
         },
+        "flowTimeoutInMinutes": {
+          "type": "string",
```

Comment:
In the code, flowTimeoutInMinutes is defined as uint?

Reply:
Also, we want to show the flowTimeoutInMinutes value at all times during a Get call.

Reply:
Why it's a string? I think it should integer

Reply:
I have updated it now.